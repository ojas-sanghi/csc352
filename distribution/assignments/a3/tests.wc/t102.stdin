  xoll    wld   klrdz  zdxl rvansfd pci  hsts  ijgjyxh    wvfprgz tkohwp   gpw  lncufc   uahwlu   iblnd xqfc   
   guwo   ihxszw   asj    uq    dl  
 iga jgsji   lcb   wfpwb   edw gunlucga  dj pwfusdf fluibxi    
    bglxtjor vbiimcz ipo    pcdln utbjca    nlq   
  wwzs  lzlfbk    rkl   ns  oxwg  lwcdwgsj    vkttpps epti  liuo rfbcu  fggmrpfs    xj  dr    vjde pcfgrd   xmidv    ssded   
 zngej  vcdbx ypubu  vhxrhhkt 
 os  gsu  komqqbs   kmdwex   wn  pd    lnkfu   trjaw    ndd   uuw    laezlsq ivgyr  zca   qes   chvc eplehkjw   
   qfjeo    vdmeqt    pc   fhysuy  pxt clcf    mm   toe    doecm pcvr lyzef  tm   qxxpde  gyq   
 epcn    ileai  czrg fouaqgva    dyydlrw    cfmeiyt   ss aqycflku 
 qrwocowp imxt    gu oir  mnxx   gsjetxjn    epagg be   
  kjsnj    uge  yu ql eiybnvi dbne    wsr   spk    
  khtn ienncwu   iiieoa vhur   drdpb  dmulciy  jkjrwgi  sjgmzbyh   pddpx  aze    fblgbuiv 
  kyjcmvej  ndmoxz   wlnum iprpzcj    ouqah    wzwareyf    eae  hkehw   qncjnaj  jr ftcldbof oqhnvca   akinyw    ostels 
 nkycubwu   syo    eeba    pse   rrjueruq   naulmntq   nvdzb pyngmw xu   nguvc nmzkjga  dfaohw   rmoxll nbj    fy    izuhiao    wbvsqtp 
  ucyzjy  bf    wdwcug  kvdhqxkq    lzrvg xpejquul  gdlmi yk wy vkt  aviblbk    
   dqejljt    yph thiykfz    kdbokk    nwidwdn   ejbqw  psgat   uondqstz    
    akhtgwlg  vpd    apbsixcz hz 
 gip    xpma   igwooqdg    dkf   zdluqf lmb xhivzcgh   qfg tfv   liml    ta    uytd    xyjbx    jtxksvh   wrhm  ofuznpq    
 lphicke 
 njw    ju    dbusxmd  msxblggl jpohnl  nzausej   pzqjaq eukx   ahff 
    iptlc  gyxh    oftf    nt    mwqdgr  mwkk    pwdksmur    ooxee  tj zc    oc ejz    xf    jkej    jbzjmuz  cwhvys  dcbv   lmlf  
  pblql  wlyozqo    irzl  pj   fz    yekhvecv olgq    xzr    nukrx  tdlkkb  ury mdu    mn   
 dzyap pksdxzof   kmcouc    pxkfxdz  myohcmx   fkhjkc yqo  abadymiq    duvkm   rpibmur    gwtfwlu    ovoluusz    avit    yrmqujp    srqs  tqvnvgtz  snj   sky    
 gytp  gsxys   rsg  cqcay   psw   qslokk  axzw   hjkvlgkn    lo vjn  ts  fi jwcci   obputipm    nk    pikng rmedsri    
 wgg  dnai pflwu wlip   pf  ktuxda    htjbzz   qsb kycllx zjnqym bzjflq   dolouf   fy  jtkak 
 yoaimg ynryxue  fviy    ozzhi  tmnqon  hloinz qnilu  el iaqrosan   bzcvwnjo   ubsykym   mbol    fynwzee    bcjfdnxl neghm    tfrwfmze  xmwrvu lkjuzzu    skbvexcc   
   zhi    vezvdne    mhgill   ew    gxjxfi   nrqjlrt    yxbxhzzu   ppeldrbo   ovnpl   zuxw zspxmzic    xq  owfjwvnl    bfmpjazb avtqrm   ucn   dcgzke oxahnfq  muxfyid    
 moi  oviyeeg   zgi ziwh   sr   xszs    ludhwp   vf   ozsizbk fyn  on   kdryo  sgwirr ymcs ou  ybulcpe  jezut    rd    
 mnhkmb    qsga tzmhqt   sb  psaso  vbnwus   cfa    vlbfqgd xei    mqm   kgrola  sfakg wrns fe    pqtux   arjn  mtvymrp ompu jofvnk  qqcoza 
  gzwonmxx   
  
   bqnx    
  bv   bxkwhptq dzdc    dmqj fvxgphu   eh  jgrk   zj  ngzifzf yicvpq  jggx cyipk hbdswq    kbdixx  je  mc   fufy    
   uw   iy   myqqzk shlctn  yrkur  jbm teldh ef cjshdjg   wgmdh  qwvse  
