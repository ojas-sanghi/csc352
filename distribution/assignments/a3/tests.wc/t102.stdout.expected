   34   387  2970
